framework/test/unit/modules/interoperability/internal_method.spec.ts

```diff
@@ -694,11 +717,15 @@ describe('Base interoperability internal method', () => {
 			const ccu = {
 				...ccuParams,
 				certificate: codec.encode(certificateSchema, certificate),
-				activeValidatorsUpdate: [
-					{ blsKey: Buffer.from([0, 0, 0, 0]), bftWeight: BigInt(5) },
-					{ blsKey: Buffer.from([0, 0, 3, 0]), bftWeight: BigInt(5) },
-					{ blsKey: Buffer.from([0, 0, 0, 1]), bftWeight: BigInt(5) },
-				],
+				activeValidatorsUpdate: {
```

Comment:
Not sure if it was already present, but I would add a case where the update contains duplicate BLS keys.